app/Singletons/OsuAuthorize.php

```diff
@@ -1905,6 +1906,46 @@ public function checkScorePin(?User $user, ScoreBest|Solo\Score $score): string
         return 'ok';
     }
 
+    public function checkTeamApplicationAccept(?User $user, TeamApplication $application): ?string
+    {
+        $this->ensureLoggedIn($user);
+
+        $team = $application->team;
+
+        if ($team->leader_id !== $user->getKey()) {
+            return null;
+        }
+        if ($team->emptySlots() < 1) {
+            return 'team.member.store.full';
```

Comment:
wrong/missing key

Also not sure if admin should be skipping the team slots check.

Reply:
I think it's fine for admin to be able to skip that 🤔 it's not like some breaking/validation thing (and it's not even a static number in the first place)